.github/workflows/release.yaml

```diff
@@ -33,3 +39,16 @@ jobs:
           REGISTRY_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           GITHUB_REPO_URL: ${{ github.server_url }}/${{ github.repository }}.git
         run: build/registry/rules-registry push-to-oci registry.yaml ${{github.ref_name}}
+
+      - name: Configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          role-to-assume: ${{ secrets.AWS_ROLE_TO_ASSUME }}
```

Comment:
Note for the reviewer: we could write this in plain text but I prefer a secret. WDYT?

Reply:
we can use a secret for now, it could be changed in the future